src/queries/query.ts

```diff
@@ -10,8 +10,10 @@ import type { VirtualCompoundConstraint } from './virtual-query/compound-query';
 import type { NormalizedAst } from '../r-bridge/lang-4.x/ast/model/processing/decorate';
 import { executeDataflowQuery } from './catalog/dataflow-query/dataflow-query-executor';
 import type { DataflowQuery } from './catalog/dataflow-query/dataflow-query-format';
+import { executeIdMapQuery } from './catalog/id-map-query/id-map-query-executor';
+import type { IdMapQuery } from './catalog/id-map-query/id-map-query-format';
 
-export type Query = CallContextQuery | DataflowQuery;
+export type Query = CallContextQuery | DataflowQuery | IdMapQuery;
```

Comment:
probably we should generate this type automatically from the `SupportedQueries`.

Reply:
Yes, this would be simpler indeed. But that's definitely not part of this PR ;)